apps/server/src/shared/infra/database/management/database-management.service.ts

```diff
@@ -60,5 +61,10 @@ export class DatabaseManagementService {
 
 	async dropCollection(collectionName: string): Promise<void> {
 		await this.db.dropCollection(collectionName);
+		await this.orm.getSchemaGenerator().createSchema();
```

Comment:
This is important to execute syncIndexes? Then we should add it to this method and not to drop or?